### DISCUSSION TITLE
Will not show my 308 solo plots after update...

### DISCUSSION THREAD

**Discussion by cdawg2020** [Chia-Network/chia-blockchain#7390] [Support]
Why wont my Plot or Farm Screen in the GUI update to show my plots...they are all solo plots...now all i see is a revolving circle like its stuck... its Synced Sooo..?   please help

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
